machines/darwin.nix

```diff
@@ -0,0 +1,4 @@
+{ config, pkgs, ... }:
+{
+  targets.genericLinux.enable = false;
```

Comment:
I think this probably would still give the same error. IIRC, it was about `targets` not existing.

Reply:
that should define `targets` though i think.